Readme.adoc

```diff
@@ -57,9 +57,31 @@ You add the dependency to the Neo4j-Java-Driver driver in your `pom.xml`:
 </dependency>
 ----
 
-To use the driver, you provide a connection URL, e.g. `bolt://user:password@localhost`, get a `Driver` instance, from which you get a Session.
+To use the driver, you provide a connection URL, e.g. `localhost`, and credentials, get a `Driver` instance, from which you get a Session.
 The session allows you to execute Cypher statements directly and iterate over the `StatementResults` to access the resulting `Value`s.
-Please edit example.movies.util.Util.java to add your connection settings, for instance "bolt://neo4j:neo4j@localhost"
+
+=== Configuration options
+
+[%header,cols=2*]
+|===
+|Environment variable name
+|Default value (or N/A)
+
+|PORT
+|8080
+
+|NEO4J_HOST
+|demo.neo4jlabs.com/movies
+
+|NEO4J_USER
+|movies
+
+|NEO4J_PASS
```

Comment:
NEO4J_PASSWORD for consistency (we should use the same env variables then everywhere)